Code/CIPLabeler/Wrap/rdCIPLabeler.cpp

```diff
@@ -0,0 +1,55 @@
+//
+//
+//  Copyright (C) 2020 Schrödinger, LLC
+//
+//   @@ All Rights Reserved @@
+//  This file is part of the RDKit.
+//  The contents are covered by the terms of the BSD license
+//  which is included in the file license.txt, found at the root
+//  of the RDKit source tree.
+//
+#include <string>
+
+#include "RDBoost/Wrap.h"
+#include "RDBoost/python.h"
+
+#include "GraphMol/RDKitBase.h"
+#include "CIPLabeler/CIPLabeler.h"
+
+namespace python = boost::python;
+using RDKit::CIPLabeler::assignCIPLabels;
+
+void assignCIPLabelsWrapHelper(RDKit::ROMol &mol,
+                               const python::object &atomsToLabel,
+                               const python::object &bondsToLabel) {
+
+  auto atoms = pythonObjectToDynBitset(atomsToLabel, mol.getNumAtoms());
+  auto bonds = pythonObjectToDynBitset(bondsToLabel, mol.getNumBonds());
+
+  // If both atoms and bonds are None, assign all the mol.
+  if (!atomsToLabel && !bondsToLabel) {
+    atoms.set();
+    bonds.set();
+  }
+
+  assignCIPLabels(mol, atoms, bonds);
+}
+
+BOOST_PYTHON_MODULE(rdCIPLabeler) {
+  python::scope().attr("__doc__") =
+      "Module containing a function to assign stereochemical labels based "
```

Comment:
I think it's worth acknowledging here in the function comment that the implementation is derived from centres (by mentioning John M and linking to his github repo)

Reply:
Done.